man/rgbasm.5

```diff
@@ -328,76 +328,100 @@ with a non-zero constant as either operand will be constant 1, even if the other
 returns 1 if the operand was 0, and 0 otherwise.
 Even a non-constant operand with any non-zero bits will return 0.
 .Ss Fixed-point expressions
-Fixed-point numbers are basically normal (32-bit) integers, which count fractions instead of whole numbers.
-They offer better precision than integers but limit the range of values.
-By default, the upper 16 bits are used for the integer part and the lower 16 bits are used for the fraction (65536ths).
-The default number of fractional bits can be changed with the
+Fixed-point numbers are basically normal integers, but conceptually they contain a dot at a fixed location\(emhence the name.
+This gives them an increased precision, at the cost of a smaller range, while remaining far cheaper to manipulate than floating-point numbers.
```

Comment:
```suggestion
This gives them increased precision at the cost of a smaller range, while remaining far cheaper to manipulate than floating-point numbers (which RGBASM does not support).
```

Reply:
`which neither .Nm nor the Game Boy support`, maybe? (I want to point out the lack of a FPU.)

Reply:
No, do not. By your idea of "does not support", the Game Boy doesn't even support multiplication!

Reply:
I think the Pan Docs are the place for discussing the Game Boy's (lack of) FPU. :P

Reply:
I fear that people are then going to ask why RGBASM doesn't support floating-point arith, instead of this weird nonsense of half-different operators.

Reply:
Which is a legitimate question! The platform where RGBASM runs certainly supports floats, so RGBASM easily could. This is a design decision, not a limitation of any kind, and there's no reason to lie to people about it.

Reply:
#956 was a serious feature request for floating-point support in RGBASM. And fixed-point support isn't something new; it's been an RGBASM feature for years. It's nice to be cleaning up the docs for it, but it's not like we're preparing a surprising and confusing new feature for people.